### ISSUE TITLE
Input type="date" polyfill (date picker): Proposition for the WET DatePicker plugin

### ISSUE THREAD

**Issue by joel-ld-esdc** [wet-boew/wet-boew#8445]
Hi, 

I would like to propose a new feature for the wet DateTime picker.
This DateTime picker is suppose to be enabled when the browser doesn't have a built in DateTime picker.
I.E. : In Internet Explorer, the Wet plugin is enabled and it's disabled in Firefox.

Couple weeks ago, we had a requirement to let a user be able to type manually into a Date field. The issue is that in Edge Browser (Windows 10), the built in date picker is enabled and we were not able to type manually.

Our idea was to take a input[text] and integrate a datepicker plugin into it. (it's a good practice on the web)

Unfortunately, there is no way to attach the WET datepicker to a input[text].
We had to use a 3rd party date picker plugin like : 
http://bootstrap-datepicker.readthedocs.io/en/latest/

http://keith-wood.name/licence.html

The issue with these plugins are not meet all the accessibility requirements.

My question is : 
Could we make the WET DatePicker compatible with input[text] field? 

Thanks, 

Joel

**Comment by duboisp** [comment]
@joel-ld-esdc are you able to join us at the Tuesday code sprint at Place du portage?

**Comment by duboisp** [comment]
Related #8073

**Comment by joel-ld-esdc** [comment]
@duboisp . Where in Place du Portage and when ?

**Comment by duboisp** [comment]
When:  Every Tuesday from 9h30 to 15h30
Where: 200 Promenade du portage, Second floor, room A014. Close to the Passport office.

**Comment by joel-ld-esdc** [comment]
I didn't found the office.

**Comment by duboisp** [comment]
There is a security guard behind the door, just ask for me Pierre Dubois. Or call me at 819-773-2881

**Comment by juleskuehn** [comment]
Good idea. The current WET datepicker also has some accessibility issues.

**Comment by duboisp** [comment]
Merci @joel-ld-esdc d'avoir participé au code sprint.

Tel que discuté, un numéro représentant la semaine (conçu spécifiquement pour votre produit) n'est pas une valeur valide tel que définie selon la spécification du [input[type=date]](https://www.w3.org/TR/html53/sec-forms.html#date-state-typedate). Une solution alternative serait l'ajout d'un champ dédié pour que l'utilisateur puissent entrer le numéro de la semaine. Par la suite tu peux surveillez l'événement "change" de ce champ afin d'initialiser la valeur du date picker.

Une autre solution, serait de créer un polyfill pour le input[type=week] si vous avez une certaine flexibilité afin d'adapter le numéro de la semaine selon un format "YYYY-W"
